### DISCUSSION TITLE
容器构建下1.0.1版本在反复查询的情况下内存占用不停上升

### DISCUSSION THREAD

**Discussion by heisedeheiheihei** [apache/doris#31449] [A - General / Q&A]
如题，1.0.1版本，be的内存占用在创建后初始时是80M左右，应用使用时一个请求多个查询的情况下，每次的内存占用都会上涨几个M，且不会自动减少，长时间累积下甚至会涨到10个G，导致整个系统的内存被吃完了
Doris新手，不确定是配置有问题，还是说正常现象，内存的占用会不会有自动清除的逻辑也不确定，希望大神们能有个解答，感谢！

**Comment by yagagagaga** [comment]
Can you upgrade to the latest version?

**Reply by heisedeheiheihei** [reply]
yes，upgrading the version is also one of our solutions.We will try to see if it is useful. tks

**Reply by heisedeheiheihei** [reply]
升级到2.0.0版本仍然存在这个问题，怀疑和信创环境有关，操作系统是欧拉，数据库使用的是达梦8
除了每次请求内存都会缓慢增长外，应用内的全量双写同步命令每次执行后Doris的内存占用都是直接增长而不下降，反复几次全量同步，每次都会增长全量数据的内存但是不会释放，看上去是直接追加了内存占用而不是清除原数据占用内存再去新增数据
目前只有这个信创环境是这样的现象，其他centos+mysql的组合环境都未出现这个问题
